manage/deploying/production/status_check.rst

```diff
@@ -0,0 +1,38 @@
+============================
+Status check
+============================
```

Comment:
On several other pages it is too long too, so I thought this may be normal, to avoid needing to change these lines when fixing a typo in a header.
See for example [manage/deploying/production/index.rst](https://raw.githubusercontent.com/plone/documentation/5.0/manage/deploying/production/index.rst)